src/main/java/org/folio/rest/impl/ModTenantAPI.java

```diff
@@ -52,11 +66,41 @@ private void executeStorageCleanUpJob(Vertx vertx, OkapiConnectionParams params)
     vertx.<Void>executeBlocking(blockingCodeHandler -> storageCleanupService.cleanStorage(params),
       cleanupAsyncResult -> {
         if (cleanupAsyncResult.failed()) {
-          logger.error("Error during cleaning file storage.", cleanupAsyncResult.cause().getMessage());
+          LOGGER.error("Error during cleaning file storage.", cleanupAsyncResult.cause().getMessage());
         } else {
-          logger.info("File storage was successfully cleaned of unused files");
+          LOGGER.info("File storage was successfully cleaned of unused files");
         }
       });
   }
 
+  private Future<List<String>> setupTestData(String script, Map<String, String> headers, Context context) {
+    try {
+      InputStream inputStream = this.getClass().getClassLoader().getResourceAsStream(script);
+
+      if (inputStream == null) {
+        LOGGER.info("Default data was not initialized: no resources found: {}", script);
+        return Future.succeededFuture();
+      }
+
+      String sqlScript = IOUtils.toString(inputStream, StandardCharsets.UTF_8.name());
+      if (StringUtils.isBlank(sqlScript)) {
```

Comment:
Is there a reason for using sql script and not TenantLoading in RMB? 
That is much cleaner and easier to use than script, that way data is in json format

Reply:
Changed to use TenantLoading